src/site/LegacyRedirectController.php

```diff
@@ -0,0 +1,81 @@
+<?hh // strict
+
+use HHVM\UserDocumentation\BuildPaths;
+use HHVM\UserDocumentation\APILegacyRedirectData;
+
+require_once(BuildPaths::APIDOCS_LEGACY_REDIRECTS);
+
+final class LegacyRedirectController extends WebController {
+  public async function respond(): Awaitable<void> {
+    $id = $this->getRequiredStringParam('legacy_id');
+    // Since the API redirects are quite specific, see if we are redirecting
+    // from there first.
+    $url = idx(APILegacyRedirectData::getIndex(), $id);
+    if ($url) {
+      throw new RedirectException($url);
+    } else {
```

Comment:
Remove `else {`, unindent the whole block. else is unneeded as the main block throws. Reduced indentation == easier to read